README.md

```diff
@@ -34,10 +33,7 @@ To retrieve the freshly generated DID:
 
 ### Requirements
 
-* Linux
-  * g++ to build native binaries
-* Mac OS X
-  * X Code 9 to build native binaries
+* Linux or Mac OS X
```

Comment:
Comment from #26:  The explicit platform requirement can be removed.  If it's not working on a supported Node.js platform, that's a bug.